content/pages/pricing.html

```diff
@@ -2,389 +2,263 @@
+        <a class="ui teal button" href="https://readthedocs.org/accounts/signup" data-analytics="community-signup">Sign up for an account</a>
       </div>
 
     </div>
-  </section>
-
-
-  <!-- Intro -->
-  <section class="ui vertical segment padded" id="intro">
-    <div class="ui container">
-
-      <div class="ui vertical segment basic padded">
-
-        <div class="ui grid center aligned">
-          <div class="column ten wide computer sixteen wide mobile">
-
-            <h3 class="ui header massive"><span class="ui header">About our platforms</span></h3>
-            <p>
-              A question our users often have is what the difference is between Read the Docs Community and Read the Docs for Business. The features available on both platforms are the same. The primary difference is the audience and use cases that are supported.
-            </p>
-
-          </div>
-        </div>
-
-      </div>
-
-      <div class="ui vertical segment basic padded center aligned">
-        <a href="https://docs.readthedocs.io/en/stable/choosing-a-site.html#choosing-between-our-two-sites" class="ui button black huge">Choosing between our two sites</a>
-      </div>
-
+  </div>
+</section>
+
+{# Compare plan features with a table #}
+<section id="compare">
+  <div class="ui padded container">
+
+    {% call pricing.feature_table() %}
+      {{ pricing.feature_row("Team management", "Control organization access with team permissions", "fa-users", on_community=False) }}
+      {{ pricing.feature_row("Email support", "Get help with your projects and documentation", "fa-message-question", None, "2 business days", "2 business days", "1 business day") }}
+      {{ pricing.feature_row("API access", "Write your own integrations", "fa-sparkles") }}
+
+      {{ pricing.header_row("Authoring", "fa-typewriter") }}
+      {{ pricing.feature_row("Unlimited projects", "Manage and build any number of projects", "fa-infinity") }}
+      {{ pricing.feature_row("Public repositories", "Clone public repositories", "fa-globe") }}
+      {{ pricing.feature_row("Private repositories", "Clone private repositories", "fa-eye-slash", on_community=False) }}
+      {{ pricing.feature_row("Pull request preview", "Preview changes in review", "fa-code-compare") }}
+
+      {{ pricing.header_row("Documentation building", "fa-gears") }}
+      {{ pricing.feature_row("Automatic building", "Build every commit", "fa-gears") }}
+      {{ pricing.feature_row("Build concurrency", "Concurrent build processes", "fa-rectangle-vertical-history fa-swap-opacity", 2, 2, 4, 6) }}
+      {{ pricing.feature_row("Notifications", "Receive notifications build events", "fa-envelope-dot") }}
+      {{ pricing.feature_row("Environment variables", "Inject secrets into builds", "fa-lock-keyhole") }}
+      {{ pricing.feature_row("Automation rules", "Automatically manage versions", "fa-robot") }}
+
+      {{ pricing.header_row("Documentation hosting", "fa-browser") }}
```

Comment:
This checklist makes it really obvious that our Pro plan isn't differentiated enough 🙃

Reply:
Yuup. And we knew that of course, but this view does make it more obvious